outputs/alertmanager_test.go

```diff
@@ -75,3 +75,66 @@ func TestNewAlertmanagerPayloadDropEvent(t *testing.T) {
 
 	require.Equal(t, o1, o2)
 }
+
+func TestNewAlertmanagerPayloadBadLabels(t *testing.T) {
+	input := `{"hostname":"host","output":"Falco internal: syscall event drop. 815508 system calls dropped in last second.","output_fields":{"ebpf/enabled":"1","n drops/buffer?clone{fork]enter":"0","n_drops_buffer_clone_fork_exit":"0"},"priority":"Debug","rule":"Falco internal: syscall event drop","time":"2023-03-03T03:03:03.000000003Z"}`
+	expectedOutput := `[{"labels":{"ebpf_enabled":"1","eventsource":"","hostname":"host","n_drops_buffer_clone_fork_enter":"0","n_drops_buffer_clone_fork_exit":"0","priority":"Warning","rule":"Falco internal: syscall event drop","severity":"warning","source":"falco"},"annotations":{"description":"Falco internal: syscall event drop. 815508 system calls dropped in last second.","info":"Falco internal: syscall event drop. 815508 system calls dropped in last second.","summary":"Falco internal: syscall event drop"},"endsAt":"0001-01-01T00:00:00Z"}]`
+	var f types.FalcoPayload
+	d := json.NewDecoder(strings.NewReader(input))
+	d.UseNumber()
+	err := d.Decode(&f) //have to decode it the way newFalcoPayload does
+	require.Nil(t, err)
+
+	config := &types.Configuration{
+		Alertmanager: types.AlertmanagerOutputConfig{DropEventDefaultPriority: Critical},
+	}
+	json.Unmarshal([]byte(defaultThresholds), &config.Alertmanager.DropEventThresholdsList)
+
+	s, err := json.Marshal(newAlertmanagerPayload(f, config))
+	require.Nil(t, err)
+
+	var o1, o2 []alertmanagerPayload
+	require.Nil(t, json.Unmarshal([]byte(expectedOutput), &o1))
+	require.Nil(t, json.Unmarshal(s, &o2))
+
+	require.Equal(t, o1, o2)
+}
+
+func Test_alertmanagerSafeLabel(t *testing.T) {
+	tests := []struct {
+		label string
+		want  string
+	}{
+		{
+			label: "host",
+			want:  "host",
+		},
+		{
+			label: "host_name",
+			want:  "host_name",
+		},
+		{
+			label: "host{name}",
+			want:  "host_name",
+		},
+		{
+			label: "host[name]",
+			want:  "host_name",
+		},
+		{
+			label: "host(name)",
+			want:  "host_name",
+		},
+		{
+			label: "json.value[/user/extra/sessionName]",
+			want:  "json_value__user_extra_sessionName",
```

Comment:
This results in a double underscore '_' not sure if this is ok. We could replace double '__' with single '_', but this would prevent and explicit use of '__'.

Reply:
I don't see any point to set use two `_`, good to me to replace by a single one.

Reply:
ok that should do it.